### PR TITLE
fix: Replace ClusterRole with Role

### PR DESCRIPTION
**Pull request by kamiiiel**
This PR is to solve this issues https://community.gravitee.io/t/unable-to-install-am-helm-chart-in-more-than-one-namespace-in-the-same-cluster/1248/1

### CODE REVIEW

**Review by jgiovaresco — APPROVED** [review]

**Comment by brasseld** [comment]
@leleueri Can you please check that one ?

File: am/templates/common/clusterrole.yaml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{{- if .Values.am.managedServiceAccount }}
+{{- if and .Values.am.managedServiceAccount .Values.am.clusterRoleRules }}
```
Comment (leleueri) [inline am/templates/common/clusterrole.yaml:1]: Do we still need clusterRole manifest ?

File: am/templates/common/clusterrolebinding.yaml [hunk 2]
```diff
@@ -1,4 +1,4 @@
-{{- if .Values.am.managedServiceAccount }}
+{{- if and .Values.am.managedServiceAccount .Values.am.clusterRoleRules }}
```
Comment (leleueri) [inline am/templates/common/clusterrolebinding.yaml:1]: Do we still need clusterRoleBinding manifest ?

**Review by leleueri — CHANGES_REQUESTED** [review]

File: am/templates/common/clusterrole.yaml (hunk 1 above)
Comment (kamiiiel) [reply am/templates/common/clusterrole.yaml:1]: We don't need it if the user wants to put everything in the same namespace but it is better to keep the cluster role option for those users who want to:
1) Have access to different resources in other namespaces
2) Resolve property files inside gravitee.yaml using KubernetesPropertyResolver from all namespaces

**Comment by kamiiiel** [comment]
It can be `managedServiceAccount: false` by deafult

File: am/templates/common/clusterrole.yaml (hunk 1 above)
Comment (leleueri) [reply am/templates/common/clusterrole.yaml:1]: in that case ok.

**Pull request merged**